pkg/config/field.go

```diff
@@ -191,6 +191,14 @@ type SetFieldConfig struct {
 	// f17elem = *f17iter.DBSecurityGroupName
 	// ```
 	From *string `json:"from,omitempty"`
+	// resources:
+	//   User:
+	//     fields:
+	//       URL:
+	//         set:
+	//           - method: Update
+	//			   to: NewURL
```

Comment:
```suggestion
	//	     to: NewURL
```

Reply:
I think it's just Github preview.. i see proper indentation from here:
<img width="558" alt="Screenshot 2023-05-02 at 11 22 51" src="https://user-images.githubusercontent.com/10897901/235726074-f36a027c-b078-4b92-aab3-dc404493c4c3.png">

Reply:
Sniper eyes!